### ISSUE TITLE
Issue Playing Custom .wav sounds for chat notifs

### ISSUE THREAD

**Issue by TinyKawaiiSatan** [chatty/chatty#401]
Hi hi, I've been using Chatty primarily for chat sound notifs, since my stream is decently small. I got real tired of the basic "ding" notification and attempted to upload my own .wav file into the directory. The sound works just fine when previewed from File Explorer, however when I try to Test the sound in Chatty I get this error:

![image](https://user-images.githubusercontent.com/30357225/106672461-122a9f80-657e-11eb-8223-33733068c5e1.png)
Caption: "Error Playing Sound.  javax.sound.sampled.UnsupportedAudioFileException: could not get audio input stream from input file"

To clarify I placed the .wav file into [Chatty>app>sounds] so that its in the same directory as the original chatty sound files. Then in the Chatty program I go to [File>Settings>Window>Notifications>Chat Message>Sound] I select my custom .wav sound from the "Sound File:" drop down menu, hit save, then hit "Test Sound." Then I get the error pictured above

What exactly is keeping my .wav from being playable through Chatty? Any idea on how to fix this?

**Debug Log**
[Include a Chatty debug log: https://chatty.github.io/help/help-report_issue.html#log ]

**Screenshots/Videos**
[If applicable, add screenshots or videos to help explain your problem.]

**Additional information (please complete the following information):**
- OS: Windows 10
- Chatty Version:  0.13.1 wind standalone

**Comment by tduva** [comment]
Does the .wav file follow the properties as suggested in the Sound Notification Settings? I don't really know what files Java supports, but changing these things could help.

![grafik](https://user-images.githubusercontent.com/5621113/106903245-111e7d00-66fa-11eb-88a1-3e2be1564233.png)

**Comment by TinyKawaiiSatan** [comment]
I will double check tonight!

> 
> 
> Does the .wav file follow the properties as suggested in the Sound Notification Settings? I don't really know what files Java supports, but changing these things could help.
> 
> ![grafik](https://user-images.githubusercontent.com/5621113/106903529-60fd4400-66fa-11eb-8819-77e3705a0f0a.png)

**Comment by TinyKawaiiSatan** [comment]
> 
> 
> Does the .wav file follow the properties as suggested in the Sound Notification Settings? I don't really know what files Java supports, but changing these things could help.
> 
> ![grafik](https://user-images.githubusercontent.com/5621113/106903529-60fd4400-66fa-11eb-8819-77e3705a0f0a.png)

![image](https://user-images.githubusercontent.com/30357225/107129569-37782000-6894-11eb-993a-227c188f6de6.png)

It's showing as 48kHz and 16bit, so according to that, it should work... but then again it says "probably".

For note, I also attempted swapping to the java version of Chatty instead of the standalone, but it made no difference.

**Comment by tduva** [comment]
Could you send me the sound file so I can test it myself?

**Comment by TinyKawaiiSatan** [comment]
[Here you go, have at it.](https://drive.google.com/file/d/1-yRb4j4vIrFoQ6c1k5XXfMGIHZw27zRL/view?usp=sharing)

**Comment by tduva** [comment]
I opened the file in Audacity and saved it again in what seemed to be the same format and the error was gone, however the file played completely silent in Chatty. It only worked when I padded it with some silence to get it over 1 second length. So it seems that it both didn't like your file in particular (even though it seems to play in other programs just fine) and also doesn't like very short files.

[kitten meow 1_fixed.zip](https://github.com/chatty/chatty/files/5940863/kitten.meow.1_fixed.zip)
